libs/designer/src/lib/ui/panel/panelTabs/parametersTab/index.tsx

```diff
@@ -354,6 +359,8 @@ const ParameterSection = ({
           onValueChange: (newState: ChangeState) => onValueChange(id, newState),
           onComboboxMenuOpen: () => onComboboxMenuOpen(param),
           pickerCallbacks: getPickerCallbacks(param),
+          onCastParameter: (value: ValueSegment[], type?: string, format?: string, suppressCasting?: boolean) =>
+            castParameterToString(value, type, format, suppressCasting),
```

Comment:
Why can't you directly pass onCastparameter: castParameterToString
The arguments look exactly the same